app/livestream/client/styles/liveStreamTab.css

```diff
@@ -1,4 +1,5 @@
 .contextual-bar__content {
+	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

Reply:
done please check